record/type.go

```diff
@@ -44,19 +64,39 @@ var AllTypeClassifications = []TypeClassification{
 // Entities are supposed to represent resources, datasets and schema.
 // XXX(Aman): should Type names be case insensitive?
 type Type struct {
-	Name           string             `json:"name"`
+	Name           TypeName           `json:"name"`
 	Classification TypeClassification `json:"classification"`
```

Comment:
CMIIW but is the reason you keep this Type struct because of `classification` field?
We can actually remove `classification` field since we are not using it anywhere else now.

Reply:
Tbh because I don't know it is important or not 😂. Ok thanks stew, will remove it.